Update start-virtual-machine-connect.md (#73132)
Azure/azure-content: articles/virtual-desktop/start-virtual-machine-connect.md

> This preview version is provided without a service level agreement, and it's not recommended for production workloads. Certain features might not be supported or might have constrained capabilities. 
> For more information, see [Supplemental Terms of Use for Microsoft Azure Previews](https://azure.microsoft.com/support/legal/preview-supplemental-terms/).

> The Start Virtual Machine (VM) on Connect (preview) feature lets you save costs by allowing end users to power on their VMs just when they need them. You can then have the VMs powered off when not required.

Reviewer: ```suggestion
The Start Virtual Machine (VM) on Connect (preview) feature lets you save costs by allowing end users to turn on their VMs only when they need them. You can then turn off VMs when they're not needed.
```
